app/src/main/java/net/squanchy/service/firebase/UnauthenticatedFirebaseDbService.java

```diff
@@ -25,23 +27,23 @@ public UnauthenticatedFirebaseDbService(DatabaseReference database) {
     }
 
     @Override
-    public Observable<FirebaseInfoItems> info() {
-        return observeChild("info", FirebaseInfoItems.class);
+    public Observable<FirebaseDays> days() {
+        return observeRoot().map(root -> new FirebaseDays(root.days));
```

Comment:
Can't you use `observeChild("newmodel/days", FirebaseDays.class)` instead of this?

If you do it this way, and "filter" data manually later, you're querying _the whole database_ every time. Not the best perf thing ever.